tools/latest-kernel-full-config.sh

```diff
@@ -213,3 +160,56 @@ validate_kernel_configs() {
         bail "Kernel configuration validation failed"
     fi
 }
+
+# Set paths
+KERNEL_KIT_DIR="/bottlerocket-kernel-kit"
+# Ensure this script is run within the Bottlerocket SDK container
+if [[ ! -d "${KERNEL_KIT_DIR}" ]]; then
+    bail "This script must be run within the Bottlerocket SDK container." \
+         "Expected mount point ${KERNEL_KIT_DIR} not found."
+fi
+
+# Check which kernels have RPMs available and process only those
+kernels_to_process=()
+for kernel_pkg in "kernel-6.1" "kernel-6.12"; do
```

Comment:
What if instead of hardcoding the list of kernels, you just generate the configs for all the kernels that you found in the packages directory? That way, we don't have to keep updating the script as we add more kernels.